src/System.Private.ServiceModel/tools/test/Bridge/Bridge/ensureBridge.ps1

```diff
@@ -24,7 +24,9 @@ $result= checkBridge;
 if(!$result)
 {
 	Write-Host Launching bridge.exe.
-	$bridgePath = Join-Path $PSScriptRoot bridge.exe
+
+	$bridgePath = Join-Path $PSScriptRoot ..\bin\bridge.exe
+
 	Start-Process $bridgePath $portNumber -WorkingDirectory $PSScriptRoot
```

Comment:
The root error was invoking ensureBridge.ps1 from its source location, not the bin location where it lives after a build.  Reverting the change to ensureBridge.ps1 and invoking it from its bin location